postpic/datahandling.py

```diff
@@ -706,6 +734,10 @@ def mean(self, axis=-1):
         return ret
 
     def _integrate_constant(self, axes=None):
+        if not self.islinear():
+            warnings.warn("Using method='constant' in integrate which is only suitable "
```

Comment:
This warning should be an exception as this method will otherwise yield a wrong result.